src/client/components/pages/parts/editor-achievements.js

```diff
@@ -125,19 +125,11 @@ class EditorAchievementTab extends React.Component {
 						<DragAndDrop name="rank3"/>
 					</div>
 					<div className="form-group">
-						<span>
-							<button
-								className="btn btn-default"
-								type="submit"
-							>
+						<span style={{marginLeft: '30px'}}>
+							<button className="btn btn-default" type="submit">
 								Update
 							</button>
-							<p
-								style={{
-									display: 'inline-block',
-									marginLeft: '10px'
-								}}
-							>
+							<p style={{display: 'inline-block', marginLeft: '10px'}}>
```

Comment:
On top of the modification above (moving `<button>` and `<p>` one level up), you can change that `<p>` into
`<span className="margin-left-1">….</span>`

Reply:
I don't think that it look goods with the change you suggested.

![image](https://user-images.githubusercontent.com/43867458/77253969-0561cd80-6c84-11ea-9088-dc3d0b13a5a1.png)

Reply:
It looks to me like the class `margin-left-1` is missing on "click badge to unset"

To clarify, what I meant was `<span className="margin-left-1">click badge to unset</span>` after the button.